src/python/pants/option/global_options.py

```diff
@@ -163,6 +163,42 @@ def register_bootstrap_options(cls, register):
         default_distdir_name = "dist"
         default_rel_distdir = f"/{default_distdir_name}/"
 
+        register(
+            "--backend-packages",
+            advanced=True,
+            type=list,
+            default=[],
+            help=(
+                "Register functionality from these backends.\n\nThe backend packages must be "
```

Comment:
The original said `Register rules from these backends.`, which leaks an implementation detail. Normal users don't need to know what a rule is.